### PR TITLE
Add support for cjs

### PR DESCRIPTION
**Pull request by DominicVonkPON**

### CODE REVIEW

File: dist/error-constructors.cjs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+var __defProp = Object.defineProperty;
```
Comment (stephantromer-pon) [inline dist/error-constructors.cjs]: Dist?

File: package.json [hunk 2]
```diff
@@ -1,29 +1,36 @@
 {
-	"name": "serialize-error",
+	"name": "@ponbike/serialize-error",
 	"version": "11.0.2",
 	"description": "Serialize/deserialize an error into a plain object",
 	"license": "MIT",
-	"repository": "sindresorhus/serialize-error",
+	"repository": "ponbike/serialize-error",
 	"funding": "https://github.com/sponsors/sindresorhus",
```
Comment (stephantromer-pon) [inline package.json:7]: Funding?

File: package.json [hunk 3]
```diff
@@ -1,29 +1,36 @@
...
+	"exports": {
+		"import": "./src/index.js",
+		"require": "./dist/index.cjs",
+		"types": "./src/index.d.ts"
+	},
 	"engines": {
 		"node": ">=14.16"
 	},
 	"scripts": {
 		"//test": "xo && ava && tsd",
```
Comment (stephantromer-pon) [inline package.json:23]: Do we still need this?

File: dist/error-constructors.cjs (hunk 1 above)
Comment (DominicVonkPON) [reply dist/error-constructors.cjs]: Removed dist

File: package.json (hunk 3 above)
Comment (DominicVonkPON) [reply package.json:23]: I also made a PR to the author that he can add the support (atleast I send a message if he can allow us to create a PR)

File: package.json (hunk 2 above)
Comment (DominicVonkPON) [reply package.json:7]: It's the author of  serialize-error so I keep it in

**Pull request merged**